charts/kubearchive/README.md

```diff
@@ -0,0 +1,60 @@
+# KubeArchive Helm Chart
+
+The kubearchive helm chart deploys the following:
+* Namespace named `kubearchive`
+* ClusterRole named `kubearchive`
+* ClusterRoleBinding named `kubearchive`
+* Service Account named `kubearchive` in the `kubearchive` namespace
+* ApiServerSource named `api-server-source` in the `kubearchive` namespace
+* Deployment and Service for `kubearchive-sink` in the `kubearchive` namespace
+* (optionally) Namespace named `test`
```

Comment:
What gets deployed and where has changed since I initially wrote this so it needs to be updated (sorry about that)

```suggestion
The kubearchive helm chart deploys the following:
* Namespace named `kubearchive`
* Service Account named `kubearchive` in the `kubearchive` namespace
* Service Account named `kubearchive-api-server` in the `kubearchive` namespace
* Deployment and Service for `kubearchive-sink` in the `kubearchive` namespace
* Deployment and Service for `kubearchive-api-server` in the `kubearchive` namespace
* (optionally) Namespace named `test`
* (optionally) Role named `kubearchive` in the `test` namespace
* (optionally) RoleBinding named `kubearchive` in the `test` namespace
* (optionally) Service Account named `kubearchive` in the `test` namespace
* (optionally) ApiServerSource named `api-server-source` in the `test` namespace
```